src/Stratis.Bitcoin.Features.Wallet.Tests/WalletControllerTest.cs

```diff
@@ -741,8 +739,8 @@ public void GetGeneralInfoSuccessfullyReturnsWalletGeneralInfoModel()
             string testWalletPath = Path.Combine(AppContext.BaseDirectory, "stratisnode", testWalletFileName);
             string folder = Path.GetDirectoryName(testWalletPath);
             var files = new string[] { testWalletFileName };
-            mockWalletManager.Setup(w => w.GetWalletsFiles()).Returns((folder, files));
-            mockWalletManager.Setup(w => w.GetWalletFileExtension()).Returns(walletFileExtension);
+            //mockWalletManager.Setup(w => w.GetWalletsFiles()).Returns((folder, files));
```

Comment:
Please remove or add comment on why this commented out.

Reply:
Restored.